.github/workflows/io-pr.yaml

```diff
@@ -0,0 +1,63 @@
+name: IO WWW PR Check
+on:
+  push:
+    branches:
+      - main
```

Comment:
do we need to trigger it on push to main?

Reply:
ahhhh i see now.. so it will run tests after push to main, but won't deploy.
tbh i find it a bit confusing but it works i guess :D